packages/playground/blueprints/src/lib/steps/login.ts

```diff
@@ -21,44 +19,23 @@ export type LoginStep = {
 	 */
 	username?: string;
 	/**
-	 * The password to log in with. Defaults to 'password'.
+	 * @deprecated The password field is deprecated and will be removed in a future version.
+	 * Only the username field is required for user authentication.
 	 */
 	password?: string;
 };
 
 /**
  * Logs in to Playground.
- * Under the hood, this function submits the [`wp-login.php`](https://developer.wordpress.org/reference/files/wp-login.php/) [form](https://developer.wordpress.org/reference/functions/wp_login_form/)
- * just like a user would.
+ * Under the hood, this function sets the `PLAYGROUND_AUTO_LOGIN_AS_USER` constant.
+ * The `auto_login.php` mu-plugin uses that constant to log in the user on the first load.
```

Comment:
Let's also document where does the `auto_login.php` mu-plugin live – it's not bundled with steps.

Reply:
It's updated now and mentiones the plugin is located in and loaded by the `@wp-playground/wordpress` package.

I didn't include the exact path as it's easy to locate in a editor, but the mention should make it clear that this feature depends on `@wp-playground/wordpress`.